README.md

```diff
@@ -60,35 +76,29 @@ You will now be able to use [`/mute`](./moderation-commands/mute.md),  [`/tempmu
 
 ## Evasion Bans
 
-Evasions bans are a fallback moderation feature to ensure muted users cannot abuse improperly configured channels. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
+**Evasion Bans required Premium. [Vote for MoonlightBot](./support/upvote-moonlightbot.md) to receive a renewable free trial of premium!**
+Evasions bans are a fallback moderation feature to ensure muted members cannot abuse improperly configured permissions by escalating a mute punishment to a harsher ban. If a user with your server's mute role sends a message in a channel that hasn't been allowed, they will be banned. To enable evasion bans, use the command
 ```
 /config settings mute-evasion-ban:True
 ```
-All channels will now be monitored for new messages sent by muted users. To create an exception to the evasion ban and allow muted users to talk in a channel, use
+All channels will now be monitored for new messages sent by muted members. To create an exception to the evasion ban and allow muted members to talk in a channel, use
 ```
 /config channels channel:IGNORED-CHANNEL ignore-mute-evasion-ban:True
 ```
-where `IGNORED-CHANNEL` is the channel you want ignored.
+where `IGNORED-CHANNEL` is the channel you want ignored. Ignoring a channel will also ignore threads made in the channel.
 
 It is recommended to set up at least one [logging channel](./README.md#logging) with the `BAN` log enabled to see when an evasion ban is triggered.
 
 ![Ban log of an evasion ban](./.gitbook/assets/EvasionBanLog.png)
 
-## Changing the Bot's Language
+To troubleshoot why the muted member was able to send a message in the unintended channel, check the following:
```

Comment:
This is too much for the home page! It could be moved onto the page for the create-muterole command